Data/Sys/Profiles/Wiimote/Wii Remote With MotionPlus Pointing.ini

```diff
@@ -0,0 +1,86 @@
+[Profile]
+Device = Bluetooth/0/Wii Remote
+Buttons/A = `A`
+Buttons/B = `B`
+Buttons/1 = `1`
+Buttons/2 = `2`
+Buttons/- = `-`
+Buttons/+ = `+`
+Buttons/Home = `HOME`
+IR/Center = 0.00 0.00
+Swing/Center = 0.00 0.00
+Tilt/Center = 0.00 0.00
+IMUAccelerometer/Up = `Accel Up`
+IMUAccelerometer/Down = `Accel Down`
+IMUAccelerometer/Left = `Accel Left`
+IMUAccelerometer/Right = `Accel Right`
+IMUAccelerometer/Forward = `Accel Forward`
+IMUAccelerometer/Backward = `Accel Backward`
+IMUGyroscope/Dead Zone = 3.0000000000000000
+IMUGyroscope/Pitch Up = `Gyro Pitch Up`
+IMUGyroscope/Pitch Down = `Gyro Pitch Down`
+IMUGyroscope/Roll Left = `Gyro Roll Left`
+IMUGyroscope/Roll Right = `Gyro Roll Right`
+IMUGyroscope/Yaw Left = `Gyro Yaw Left`
+IMUGyroscope/Yaw Right = `Gyro Yaw Right`
+IMUIR/Enabled = True
+IMUIR/Total Yaw = 20.000000000000000
+Extension/Attach MotionPlus = `Attached MotionPlus`
+Extension = `Attached Extension`
+Nunchuk/Buttons/C = `Nunchuk C`
+Nunchuk/Buttons/Z = `Nunchuk Z`
+Nunchuk/Stick/Up = `Nunchuk Y+`
+Nunchuk/Stick/Down = `Nunchuk Y-`
+Nunchuk/Stick/Left = `Nunchuk X-`
+Nunchuk/Stick/Right = `Nunchuk X+`
+Nunchuk/Stick/Calibration = 99.03 96.27 97.29 102.30 108.95 101.86 99.22 100.30 105.52 100.70 99.97 101.77 103.68 99.30 95.07 94.67 98.02 93.32 92.44 95.21 97.31 91.13 88.89 90.15 95.15 92.96 92.37 95.43 100.26 96.04 93.30 94.24
```

Comment:
I think it would be cleaner if you set these three "Calibration" entries to "100 100 100 100 100 100 100 100".
That should produce a perfect octagon to match up with the calibration adjusted values from the Nunchuk/CC.